app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -135,16 +135,30 @@ def stop_middleware_server(ems_ref)
       run_generic_operation(:Shutdown, ems_ref)
     end
 
-    def stop_middleware_deployment(ems_ref)
+    def stop_middleware_deployment(ems_ref, name)
```

Comment:
It's not used other than in the undeploy, or you could say it's reserved for future use, I guess. All of the operation entry points are called from the generic op.call(ems_ref, name), and I needed it for undeploy. Perhaps in Ruby I could just not declare it on the methods that don't use it?  Does Ruby care if you pass in an unused param at the end?